### PR TITLE
Implement reader tags for java.time.Instant

### PR DESCRIPTION
**Pull request by plexus**
Also make sure the stuff in user.clj only gets loaded on demand.

@barrosfelipe  can you review and merge? thanks!

### CODE REVIEW

File: src/activities/system.clj [hunk 1]
```diff
@@ -14,6 +14,8 @@
             [buddy.auth.middleware :refer [wrap-authentication]]
             [buddy.auth.backends :as backends]))
 
+(require 'activities.time)
+
```
Comment (loopified) [inline src/activities/system.clj:18]: Is there a reason for requiring it outside the `ns` declaration?

**Review by loopified — COMMENTED** [review]
Thank you!

File: src/activities/system.clj (hunk 1 above)
Comment (plexus) [reply src/activities/system.clj:18]: yes. it's being required for side-effects, so it's not actually used in this namespace. This means that some tools like `clj-refactor` might think it's not used and thus remove it from the `ns` form. doing it this way makes it clear that it's loaded for side effects and prevents that problem.

**Comment by loopified** [comment]
Oh, excellent. Learned a new trick :)

**Pull request merged**